### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by basalicante** [dusanst/DynamicSpriteSheets#3]
Hi guys,

This looks awesome and it's exactly what I need, but I've got no idea how to implement the actual code.

I can create a new GridAtlas and add Textures to it. But then how do I retrieve them? Looks like it should be through the Atlas class, but I've got no idea how to connect the Atlas to the GridAtlas. Or however this should work.

Would you be so kind as to make a minimal example code? Even if it's just a couple of lines showing how the basic structure works. That would be so helpful!

Cheers,
bas

**Comment by antpaw** [comment]
true, hope someone can do that, couple lines of high level code would be enough.

**Comment by ValentynKhylchuk** [comment]
Maybe it will help somebody.
https://www.youtube.com/watch?v=IXH46xdSUEE&ab_channel=Unity
